locale_update.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python
+
+##
+# This script is designed to update the l10n files and locale registration
+# for the standalone loop client. The source of the localization files
+# is https://github.com/mozilla/loop-client-l10n repository.
+# The loop client repo is assumed to be https://github.com/mozilla/loop-client.
+#
+# Run this script from the local version of loop-client. It assumes that a local
+# version of loop-client-l10n is in a parallel directory: ../loop-client-l10n.
+# Changes are then pushed back to loop-client.
+##
+
+import sys
+import os
+import shutil
+import io
+import re
+
+l10n_src = "../loop-client-l10n/l10n/"
+content_dir = "content/"
+index_file_name = "index.html"
+
+def createLocale(dst_name_list, src_dir):
```

Comment:
nit: PEP8 asks for `create_locale` rather than `createLocale` (which is what we would do in js)